app/src/main/java/mozilla/lockbox/view/ItemListFragment.kt

```diff
@@ -202,4 +203,15 @@ class ItemListFragment : CommonFragment(), ItemListView {
         // scroll list view to top
         scrollToTop()
     }
-}
\ No newline at end of file
+
+    override fun loading(isLoading: Boolean) {
+        if (isLoading) {
+            showAndRemove(view!!.loadingView, view!!.entriesView)
+        } else {
+            showAndRemove(view!!.entriesView, view!!.loadingView)
+        }
+        view!!.filterButton.isClickable = !isLoading
```

Comment:
This could be a follow-up improvement, but it would be nice for these `isClickable` and `isEnabled` settings to each be their own `Consumer`s on the `View` rather than side effects of showing / hiding the loading spinner. IIRC from iOS, we have other situations (no saved entries comes to mind) where we want to disable the filter / sort buttons. However, I'm happy to let this PR merge as-is and do that work in later tickets :)

Reply:
@sashei You're right. I will merge this and add the modifications in another PR. Thanks for the review!

Reply:
Filed a follow-up issue here: https://github.com/mozilla-lockbox/lockbox-android/issues/279